apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -8,14 +8,14 @@ import {
     Grid,
     IconButton,
     InputLabel,
+    TableContainer as MUITableContainer,
```

Comment:
Why is this being moved? It looks like your formatter is changing the alphabetization rules. Fix this before checking it in.